pyairtable/utils.py

```diff
@@ -10,7 +10,9 @@ def datetime_to_iso_str(value: datetime) -> str:
     Args:
         value: datetime object
     """
-    return value.isoformat(timespec="milliseconds") + "Z"
+    if value.utcoffset() is None:
+        return value.isoformat(timespec="milliseconds") + "Z"
+    return value.isoformat(timespec="milliseconds")
```

Comment:
```suggestion
    # Add local timezone if naiv datetime object was passed in
    # https://docs.python.org/3/library/datetime.html#datetime.datetime.astimezone
    if value.tzinfo is None or value.tzinfo.utcoffset(d) is None:
        value = value.astimezone()

    return value.isoformat(timespec="milliseconds")
```

Reply:
This looks good. Thanks. Shouldn't the condition evaluate the truth of `value.tzinfo.utcoffset(value) is None`?

Reply:
Ooooh yes. good catch! I've corrected that